packages/sanity/src/core/form/inputs/files/ImageToolInput/ImageToolInput.tsx

```diff
@@ -135,10 +135,14 @@ export function ImageToolInput(props: ImageToolInputProps) {
                 <LoadStatus>
                   {imageLoadError ? (
                     <Card padding={4} radius={2} tone="critical" border>
-                      <Text>Error: {imageLoadError.message}</Text>
+                      <Text>
+                        {t('inputs.files.imagetool.field.load-error', {
```

Comment:
What does the `field` subkey represent here?

Reply:
Not sure. The title and description keys were already done by @bjoerge and I just did the error message and loading text so I followed the same naming convention.

Reply:
I think I might named it so because it's inside this component's `<FormField>`?  But, yeah, removing the `field` portion here is probably for the better!

Reply:
ok, fixed